api/src/main/java/org/pmiops/workbench/access/AccessTierServiceImpl.java

```diff
@@ -0,0 +1,55 @@
+package org.pmiops.workbench.access;
+
+import com.google.common.collect.ImmutableList;
+import java.util.Collections;
+import java.util.List;
+import java.util.stream.Collectors;
+import java.util.stream.StreamSupport;
+import javax.inject.Provider;
+import org.pmiops.workbench.config.WorkbenchConfig;
+import org.pmiops.workbench.db.dao.AccessTierDao;
+import org.pmiops.workbench.db.model.DbAccessTier;
+import org.pmiops.workbench.db.model.DbUser;
+import org.pmiops.workbench.model.DataAccessLevel;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
+
+@Service
+public class AccessTierServiceImpl implements AccessTierService {
+
+  private final Provider<WorkbenchConfig> configProvider;
+
+  private final AccessTierDao accessTierDao;
+
+  private final String REGISTERED_TIER_SHORT_NAME = "registered";
+
+  @Autowired
+  public AccessTierServiceImpl(
+      Provider<WorkbenchConfig> configProvider, AccessTierDao accessTierDao) {
+    this.configProvider = configProvider;
+    this.accessTierDao = accessTierDao;
+  }
+
+  /**
+   * A placeholder implementation until we complete the Controlled Tier access modules.
+   *
+   * <p>For registered users, return the registered tier or all tiers if we're in an environment
```

Comment:
For the moment we still depend on `user.getDataAccessLevelEnum()` as the source of truth for this information